eng/scripts/appium-install.ps1

```diff
@@ -198,8 +198,8 @@ if ($IsWindows) {
     appium driver doctor windows || & { "ignore failure"; $global:LASTEXITCODE = 0 }
 }
 if ($IsMacOS) {
-    appium driver doctor xcuitest || & { "ignore failure"; $global:LASTEXITCODE = 0 }
-    appium driver doctor mac2 || & { "ignore failure"; $global:LASTEXITCODE = 0 }
+    # appium driver doctor xcuitest || & { "ignore failure"; $global:LASTEXITCODE = 0 }
```

Comment:
Why comment these lines? Aren't these Appium drivers necessary?

Reply:
this is the doctor not the driver